src/components/FileDirectory.vue

```diff
@@ -38,6 +39,7 @@
 </template>
 
 <script>
+/* eslint-disable */
```

Comment:
Why disable lint here?

Reply:
That's a good question that I don't have a good answer for. Removed